designs/disruption-controls-by-reason.md

```diff
@@ -0,0 +1,365 @@
+# Disruption Controls By Reason
+# Table of Contents
```

Comment:
Does using `[[_TOC_]]` work so we don't have to make sure this constantly aligns with the naming/titles below? See: https://stackoverflow.com/questions/47154661/how-can-i-create-a-table-of-content-in-gitlab-wiki

Reply:
https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/ from what I can see github doesn't natively support generating them and trying the `[[_TOC_]]` syntax but it did not generate what we are looking for. 


For now, we can remove the TOC, or I can regenerate it with a separate tool once we are ready to merge this in.